tasks/us/census/tiger.py

```diff
@@ -664,10 +666,10 @@ def populate(self):
         session = current_session()
         stmt = ('INSERT INTO {output} '
                 'SELECT geoid, ST_Union(ST_MakePolygon(ST_ExteriorRing(the_geom))) AS the_geom, '
-                '       MAX(aland) AS aland, cdb_observatory.FIRST(name) AS name '
+                '       MAX(aland) AS aland '
```

Comment:
It looks like yes. Following the stream of changes between John and you I came to the conclusion, hopefully not wrong, that this is some kind of remain from that changes.

I wasn't able to check that property anywhere and was making the task fail. Also, checking the production tables we don't have a name column.